control/trajectory_follower_nodes/design/longitudinal_controller-design.md

```diff
@@ -148,7 +148,7 @@ AutonomouStuff Lexus RX 450h for under 40 km/h driving.
 
 | Name                                 | Type   | Description                                                                                                                                                                             | Default value |
 | :----------------------------------- | :----- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
-| control_rate                         | double | control rate [Hz]                                                                                                                                                                       | 30            |
+| longitudinal_ctrl_period             | double | longitudinal control period [s]                                                                                                                                                         | 0.03          |
```

Comment:
A period of 0.03s corresponds to a period of ~33Hz which is 10% higher than the previous value.
It probably does not make a big difference but I think it is better to put a period of `0.0333` to minimize the difference with the previous default value.
What do you think ?